pravega/src/test/resources/logback-test.xml

```diff
@@ -23,6 +23,7 @@
     <logger name="akka.stream.alpakka.pravega" level="debug" />
     <logger name="docs.scaladsl" level="debug" />
     <logger name="docs.javadsl" level="debug" />
+    <logger name="PravegaTableSourceStageLogic(akka://PravegaSpec)" level="debug" />
```

Comment:
Did a FQCN not work here?

Reply:
I've made some attempt but I'm confused with StageLogging :'(